include/osquery/config/packs.h

```diff
@@ -0,0 +1,101 @@
+/*
+ *  Copyright (c) 2014, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#pragma once
+
+#include <map>
+#include <memory>
+#include <string>
+#include <vector>
+
+#include <boost/property_tree/ptree.hpp>
+
+#include <osquery/database.h>
+#include <osquery/flags.h>
+#include <osquery/registry.h>
+#include <osquery/status.h>
+
+namespace osquery {
+
+typedef boost::property_tree::ptree ptree;
```

Comment:
I don't want to alias `boost::property_tree` in the public header, so having it in here allows us to use `osquery::ptree` as the symbol.